src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -16,6 +16,7 @@
 using System.Reflection.Metadata.Ecma335;
 using System.Text;
 using System.Threading;
+using Basic.Reference.Assemblies;
```

Comment:
This is indeed necessary. The removal of `Net461` from `TestBase` was replaced with the type from the NuPkg and this namespace is required to get to it.